gdx/src/com/badlogic/gdx/ApplicationListener.java

```diff
@@ -29,7 +31,7 @@
  * </p>
  * 
  * @author mzechner */
-public interface ApplicationListener {
+public interface ApplicationListener implements Disposable {
```

Comment:
You need to use `extends` for interfaces that "inherit" from another interface, not `implements`. That you didn't bother to build this, or even see if a red underline shows up in your IDE, does not instill confidence in this PR.

Reply:
This was a tiny change, I used simple text editor to do that. But forgot, that interfaces use "extends" keyword instead of "implements" for other interfaces.